### DISCUSSION TITLE
Saturation Beta in Michaelis-Menten

### DISCUSSION THREAD

**Discussion by glevv** [pymc-labs/pymc-marketing#1040] [Q&A] [answered]
I can't find this parameter. Is it a bug or am I missing something?

Also if I try to pass 'hill_sigmoid' as a string to saturation parameter, it raises an error that is an invalid option.

**Comment by williambdean** [comment, marked as answer]
Hi 

The alpha parameter in menten acts as the beta. See the formula here
https://www.pymc-marketing.io/en/stable/api/generated/pymc_marketing.mmm.transformers.michaelis_menten.html#pymc_marketing.mmm.transformers.michaelis_menten

I believe the string for hill is just "hill". 
However, we are deprecating strings in favor of passing the whole class as addition configuration needs to be stored off. See the example in the readme
